src/components/modules/IconsHome/RightIconDetailsSection/IconActionsButton.tsx

```diff
@@ -59,7 +60,7 @@ export const IconActionsButton: FC<{
               <div className="mr-2">{inlineIconsMap[iconActions[0].icon]}</div>
               <div>{iconActions[0].name}</div>
               &nbsp;
-              <span className="text-xs">(⌘⇧C)</span>
+              <span className="text-xs">({getCopyShortcut()})</span>
```

Comment:
I was thinking if we could re-use a single fn everywhere.
something like
```js
<span className="text-xs">({
platformBasedText({
  mac: '⌘⇧C,
  win: 'Ctrl+Shift+c,
  linux: 'Ctrl+Shift+c,
 })
 }</span>
```

platformBasedText util function selects the relevant option based on OS.